lib/ace/ext/papeeria/spellchecker.coffee

```diff
@@ -1,16 +1,16 @@
-define (require, exports, module) ->
-    "use strict";
+define ((require, exports, module) ->
```

Comment:
... and now when arguments are surrounded with braces pls make this code looking syntactically correct, that is, remove space between function name and arguments list.